py3status/modules/sysdata.py

```diff
@@ -7,10 +7,12 @@
     format: output format string
         *(default '[\?color=cpu CPU: {cpu_usage}%], '
         '[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]')*
-    padding: length of space padding to use on the left
-        (default 0)
-    precision: precision of values
-        (default 2)
+    format_percent: format for the values in percent
+        (default "{value:.2f}")
+    format_temp: format for the temperature value
+        (default "{value:.2f}{unit}")
+    format_value: format for the flat values
```

Comment:
Sure. I'll wait for #520 to be in and change it to `format_mem` and move `{mem_unit}` from `format` to `format_mem`.
@pferate you'd be ok with that?